src/server/config.js

```diff
@@ -5,10 +5,15 @@ let inputConfig = {};
 if (process.env.GUPPY_CONFIG_FILEPATH) {
   const configFilepath = process.env.GUPPY_CONFIG_FILEPATH;
   inputConfig = JSON.parse(readFileSync(configFilepath).toString());
-  log.info('[config] read guppy config from', configFilepath, JSON.stringify(inputConfig, null, 4));
+  log.info(
+    '[config] read guppy config from',
+    configFilepath,
+    JSON.stringify(inputConfig, null, 4),
+  );
 }
 
 const config = {
+  allowRefresh: inputConfig.allowRefresh || false,
```

Comment:
`allowRefresh` is duplicated in this config

Reply:
fixed